src/__tests__/type.js

```diff
@@ -715,3 +715,77 @@ test('should give error if we are trying to call type on an invalid element', as
     `"the current element is of type BODY and doesn't have a valid value"`,
   )
 })
+
+test('navigation key: {arrowleft} and {arrowright} moves the cursor', () => {
+  const {element, getEventSnapshot} = setup('<input />')
+  userEvent.type(element, 'Brea{arrowleft}k{arrowright}fast')
+  expect(getEventSnapshot()).toMatchInlineSnapshot(`
+    Events fired on: input[value="Brekafast"]
+
+    input[value=""] - pointerover
+    input[value=""] - pointerenter
+    input[value=""] - mouseover: Left (0)
+    input[value=""] - mouseenter: Left (0)
+    input[value=""] - pointermove
+    input[value=""] - mousemove: Left (0)
+    input[value=""] - pointerdown
+    input[value=""] - mousedown: Left (0)
+    input[value=""] - focus
+    input[value=""] - focusin
+    input[value=""] - pointerup
+    input[value=""] - mouseup: Left (0)
+    input[value=""] - click: Left (0)
+    input[value=""] - keydown: B (66)
+    input[value=""] - keypress: B (66)
+    input[value="B"] - input
+      "{CURSOR}" -> "B{CURSOR}"
+    input[value="B"] - keyup: B (66)
+    input[value="B"] - keydown: r (114)
+    input[value="B"] - keypress: r (114)
+    input[value="Br"] - input
+      "B{CURSOR}" -> "Br{CURSOR}"
+    input[value="Br"] - keyup: r (114)
+    input[value="Br"] - keydown: e (101)
+    input[value="Br"] - keypress: e (101)
+    input[value="Bre"] - input
+      "Br{CURSOR}" -> "Bre{CURSOR}"
+    input[value="Bre"] - keyup: e (101)
+    input[value="Bre"] - keydown: a (97)
+    input[value="Bre"] - keypress: a (97)
+    input[value="Brea"] - input
+      "Bre{CURSOR}" -> "Brea{CURSOR}"
+    input[value="Brea"] - keyup: a (97)
+    input[value="Brea"] - keydown: ArrowLeft (37)
+    input[value="Brea"] - select
```

Comment:
I am thinking a select event should show the before and after of the cursor position:

```
    input[value="Brea"] - keydown: ArrowLeft (37)
    input[value="Brea"] - select
      "Brea{CURSOR}" -> "Bre{CURSOR}a"
    input[value="Brea"] - keyup: ArrowLeft (37)
```

What do y'all think?

Reply:
For posterity's sake: there was an earlier comment and it got [1 response](https://github.com/testing-library/user-event/pull/404#discussion_r456944718). 

> I'm not sure. Based on the playground we shouldn't fire a select event at all I think (though it may be unfortunately unavoidable).

I made a new comment thread to leave the unexpected `{CURSOR}` position bits out of this.

Reply:
I agree